.azure-pipelines/mgmt.yml

```diff
@@ -0,0 +1,24 @@
+pr:
+  branches:
+    include:
+    - dev
+
+pool:
+  vmImage: 'VS2017-Win2016'
+
+steps:
+  - task: CmdLine@1
+    inputs:
+      filename: C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\sn.exe
+      arguments: -Vr *,31bf3856ad364e35
+  - task: MSBuild@1
+    inputs:
+      solution: build.proj
+  - task: MSBuild@1
+    inputs:
+      solution: build.proj
+      msbuildArguments: /t:RunTests /clp:ShowTimestamp /flp:Summary;Verbosity=minimal;LogFile=msbuild.sum /flp1:warningsonly;logfile=msbuild.wrn /flp2:errorsonly;logfile=msbuild.err /p:IgnorePathTokens="Microsoft.Rest.ClientRuntime.Tracing.Tests Microsoft.Azure.Services.AppAuthentication.IntegrationTests KeyVault\dataPlane LocationBasedServices Az.Auth.Tests BotService.Tests Search Batch.FileStaging.Tests Azure.Batch.IntegrationTests Azure.Batch.ProtocolTests Batch\Support\FileConventions AzureBatchFileConventions.IntegrationTests Batch\DataPlane\Azure.Batch.Unit.Tests Batch\DataPlane\Tools\ConfigureAwaitAnalyzer\ConfigureAwaitAnalyzer.Tests ContainerService\ContainerService.Tests PolicyInsights\Management\PolicyInsights.Tests Consumption\Consumption.Tests Billing\Billing.Tests ApplicationInsights\Management\ApplicationInsights.Tests Batch\Management\Management.Batch.Tests SecurityCenter\SecurityCenter.Tests AzureStack\Admin\FabricAdmin\Fabric.Admin.Tests IotHub\IotHub.Tests Monitor\Monitor.Tests"
+  - task: PublishBuildArtifacts@1
```

Comment:
Are you attempting to publish the test results? If so I'd suggest using the PublishTestResults see https://github.com/Azure/azure-sdk-for-net/blob/master/.azure-pipelines/client.yml#L135. Just need to make sure the test results format is matching and it will show up in the test tab of the build.